### PR TITLE
Fix for analising paginated pull requests result.

### PR DESCRIPTION
**Pull request by OlgaMaciaszek**
Only pull requests returned by stash in the first page were fetched. This should fix it.

### CODE REVIEW

File: src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/stash/StashPullRequestResponse.java [hunk 1]
```diff
@@ -17,6 +17,10 @@
 
     private int size;//
 
+    private Boolean isLastPage;
+
+    private Integer nextPageStart;
```
Comment (szpak) [inline src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/stash/StashPullRequestResponse.java]: Could those two fields be `null`?

Comment (OlgaMaciaszek) [inline src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/stash/StashPullRequestResponse.java]: I think  `isLastPage` could only be null if the response we receive from Stash API is incorrect, and `nextPageStart` could only be null if `isLastPage` is true. But such a case is handled in `getPullRequests()`.

Comment (szpak) [inline src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/stash/StashPullRequestResponse.java]: So maybe it should be changed to `boolean`/`int` respectively.

**Comment by jamieomatthews** [comment]
:+1:

**Comment by szpak** [comment]
Any plans to make it merged?

**Comment by nemccarthy** [comment]
LGTM

**Comment by OlgaMaciaszek** [comment]
Thanks, @nemccarthy . When can we expect a release with this change?

**Comment by nemccarthy** [comment]
This has been released in 1.4.0

**Pull request merged**